sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleJoins.java

```diff
@@ -152,6 +152,10 @@ public void setContactJoins(ContactJoins contactJoins) {
 		this.contactJoins = contactJoins;
 	}
 
+	public CaseJoins getContactCaseJoins() {
+		return getContactJoins().getCaseJoins();
```

Comment:
this added method is not used

Reply:
I will remove in the next sample dashboard related ticket I'm working on right now